atd-vze/src/Components/Notes/Notes.js

```diff
@@ -30,7 +30,7 @@ const Notes = ({
   // declare mutation functions
   const [addNote] = useMutation(INSERT_NOTE);
   const [editNote] = useMutation(UPDATE_NOTE);
-  const [deleteNote] = useMutation(DELETE_NOTE);
+  const [deleteNote] = useMutation(SOFT_DELETE_NOTE);
```

Comment:
<img width="556" alt="Screenshot 2024-09-05 at 11 09 23 AM" src="https://github.com/user-attachments/assets/644849d7-d8ca-4633-9c01-23967b0a3c62">

It's not quite right that we show the "Are you sure?" text and note text with the same styling. What do you think about not showing the note text at all in the modal?

Reply:
yeah i am very down to remove the note text completely!

Reply:
thanks, Rose! 🚢